### DISCUSSION TITLE
Formato Torpedo

### DISCUSSION THREAD

**Discussion by bozzimuc** [IIC2133-PUC/2024-2#130] [Preguntas Materia] [answered]
Buenas tardes, tengo 2 preguntas sobre el Torpedo:
1. ¿Puede ser en papel oficio (21,6 x 33 cm)?
2. ¿Puede ser en lápiz mina, o debe ser en lápiz pasta?
Otra pregunta, ¿la i1 es el lunes o el miércoles? Distintas fuentes dicen cosas distintas
Muchas gracias

**Comment by CataAlamos** [comment, marked as answer]
Hola! Tiene que ser tamaño carta o el papel de un cuaderno, la puedes escribir como tu quieras, y la I1 es el miércoles. Mandamos un anuncio por Canvas con todas las fechas.

**Reply by bozzimuc** [reply]
Muchas gracias! Entonces tengo que recortar la hoja oficio a carta?

**Reply by domiSoto** [reply]
Hola! El anuncio de canvas decía tamaño carta u oficio, al final solo puede ser carta?

**Reply by CataAlamos** [reply]
Hola! Toda la razón, guíense por el anuncio de Canvas.
